src/main/resources/static/openapi/EducationAndWorkPlanAPI.yml

```diff
@@ -1832,6 +1863,8 @@ components:
       title: NoteResponse
       type: object
       description: An object representing a note.
+      allOf:
```

Comment:
Based on earlier PR comment, the new type `ActionPlanReviewsResponse` now returns the review note as a `NoteResponse` typed field 👍 
Whilst I was here checking what fields would be returned by doing this, I noticed that `NoteResponse` had all 8 audit fields (createdBy etc), but they were explicitly listed. By using the `allOf` syntax we can include those fields from `AuditedResource`, and save some lines of code (which is a good thing because this spec is getting looooooong!)